### PR TITLE
Update KafkaSource documentation with the new installation instructions

### PR DESCRIPTION
**Pull request by pierDipi**
We're moving the KafkaSource implementation
to a new repository.

This PR changes the installation instructions and 
changes the text to verify that the KafkaSource
is ready/running.

## Proposed Changes 

- Update KafkaSource documentation with the new installation instructions

### CODE REVIEW

File: docs/eventing/sources/kafka-source/README.md [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
 
 The `KafkaSource` reads all the messages, from all partitions, and sends those messages as CloudEvents through HTTP to its configured `sink`. The `KafkaSource` supports an ordered consumer delivery guaranty, which is a per-partition blocking consumer that waits for a successful response from the CloudEvent subscriber before it delivers the next message of the partition.
 
-!!! note
-    If you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets, you can implement a Kafka Consumer by using one of the available Apache Kafka SDKs.
-
 <!--TODO: Check if this note is out of scope; should we not mention anything beyond the direct Knative/Kafka integration we provide?-->
 
-## Install the Kafka event source CRD
+## Installation
```
Comment (abrennan89) [inline docs/eventing/sources/kafka-source/README.md:11]: I'd keep it as "Installing Kafka source" maybe?

File: docs/eventing/sources/kafka-source/README.md [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+    ```
+
+## Create a Kafka topic
+
+!!! note
+    The create a Kafka topic section assumes you're using Strimzi to operate Apache Kafka,
+    however equivalent operations can be replicated using the Apache Kafka CLI or any
+    other tool.
 
 If you are using Strimzi:
```
Comment (abrennan89) [inline docs/eventing/sources/kafka-source/README.md:46]: If there's no plan to add instructions for other things I'd simply change this section to list Strimzi as a prerequisite

**Review by abrennan89 — APPROVED** [review]
Couple of minor nits but looks good!
/lgtm
/hold
Adding a hold until @matzew has a chance to review, but once ya'll are happy with it feel free to remove the hold.

File: docs/eventing/sources/kafka-source/README.md (hunk 1 above)
Comment (pierDipi) [reply docs/eventing/sources/kafka-source/README.md:11]:
```suggestion
## Installing Kafka source
```

**Comment by csantanapr** [comment]
@pierDipi your done with your changes and this is ready to merge now?

File: docs/eventing/sources/kafka-source/README.md (hunk 2 above)
Comment (matzew) [reply docs/eventing/sources/kafka-source/README.md:46]: I think that section is not related to this PR, but the below tasks about specifics to the strimzi distribution of apache kafka  - I think that Kafka as a more generic thing can be seen as a prerequisite.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

/unhold

Thanks for putting this together!

**Pull request merged**